src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
The grid-template-columns property is used incorrectly. You are repeating 200px 1 time which means you are setting the width of the column to be 200px. But this is not a correct way to set the width of the columns in a grid. Instead, you need to specify the number of columns you want to have in your grid and use the 'fr' unit to distribute the space equally among the columns. For example, 'grid-template-columns: repeat(4, 1fr);' would create a grid with 4 equal-width columns.